### ISSUE TITLE
Version 2.6.2: Organizer Points Incorrectly Calculated

### ISSUE THREAD

**Issue by ercousin** [geoffhumphrey/brewcompetitiononlineentry#1484]
While working on Brew Slam 2023 I noticed that Organizer points were calculating as 6.0 when it should have been 5.0

![image](https://github.com/geoffhumphrey/brewcompetitiononlineentry/assets/1551339/f4af0e4a-a1d5-4c8a-be5d-aeb40210719f)

We had 517 entries registered, 501 paid, 488 rec'+judged. This 488 number is what should have been used to determine 5.0 as the max organizer points.

https://www.bjcp.org/about/reference/experience-point-award-schedule/
![image](https://github.com/geoffhumphrey/brewcompetitiononlineentry/assets/1551339/9e43fd47-615c-4b37-8c01-657a9902bc3e)

**Comment by geoffhumphrey** [comment]
Eric, thanks for reporting! Confirmed. 

In [this commit](https://github.com/geoffhumphrey/brewcompetitiononlineentry/commit/b6008d0ca6b2f2253647797872057c16b3ade6a0), I've adjusted the amount of entries to base the calculations upon to be the amount marked as _received_ - the thinking being that if an entry is marked as received, it will be judged. This adjustment will make for a more accurate calculation for organizer points.
